### ISSUE TITLE
Format argument does not work for listing repos ....

### ISSUE THREAD

**Issue by taherv** [piotrmurach/github_cli#25]
Beginning ruby user, so please be patient with me.
I can't figure out a way to change the output format .... the example in the README does not work for me. Is there some trick to passing the --format parameter I don't know about ?
Thanks for your time.

``` bash
test-user@test:~/gcli/github_cli-0.6.2/bin$ ./gcli repo ls -u wycats --format=csv
Response Status: 200

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ id                        ┃ 1119731                                                                                               ┃
┃ name                      ┃ abbot-from-scratch                                                                                    ┃
┃ full_name                 ┃ wycats/abbot-from-scratch                                                                             ┃
┃ owner_login               ┃ wycats                                                                                                ┃
┃ owner_id                  ┃ 4                                                                                                     ┃
┃ owner_avatar_url          ┃ https://avatars.githubusercontent.com/u/4?v=3                                                         ┃
┃ owner_gravatar_id         ┃                                                                                                       ┃
┃ owner_url                 ┃ https://api.github.com/users/wycats                                                                   ┃
┃ owner_html_url            ┃ https://github.com/wycats                                                                             ┃
┃ owner_followers_url       ┃ https://api.github.com/users/wycats/followers                                                         ┃
┃ owner_following_url       ┃ https://api.github.com/users/wycats/following{/other_user}                                            ┃
┃ owner_gists_url           ┃ https://api.github.com/users/wycats/gists{/gist_id}                                                   ┃
┃ owner_starred_url         ┃ https://api.github.com/users/wycats/starred{/owner}{/repo}                                            ┃
┃ owner_subscriptions_url   ┃ https://api.github.com/users/wycats/subscriptions                                                     ┃
┃ owner_organizations_url   ┃ https://api.github.com/users/wycats/orgs                                                              ┃
┃ owner_repos_url           ┃ https://api.github.com/users/wycats/repos                                                             ┃
┃ owner_events_url          ┃ https://api.github.com/users/wycats/events{/privacy}                                                  ┃
┃ owner_received_events_url ┃ https://api.github.com/users/wycats/received_events                                                   ┃
┃ owner_type                ┃ User                                                                                                  ┃
┃ owner_site_admin          ┃ false                                                                                                 ┃
┃ private                   ┃ false                                                                                                 ┃
┃ html_url                  ┃ https://github.com/wycats/abbot-from-scratch                                                          ┃
┃ description               ┃                                                                                                       ┃
┃ fork                      ┃ false                                                                                                 ┃
┃ url                       ┃ https://api.github.com/repos/wycats/abbot-from-scratch                                                ┃
┃ forks_url                 ┃ https://api.github.com/repos/wycats/abbot-from-scratch/forks                                          ┃
┃ keys_url                  ┃ https://api.github.com/repos/wycats/abbot-from-scratch/keys{/key_id}                                  ┃
┃ collaborators_url         ┃ https://api.github.com/repos/wycats/abbot-from-scratch/collaborators{/collaborator}                   ┃
{code}
```

**Comment by piotrmurach** [comment]
Hi Taher, great that you gave this library a go and sorry for late reply.

Currently I'm rewriting this library and updating it to use latest GitHub api which will take some time. The library hasn't been released for a long time and hence I would expect the `format` option to be broken.

**Comment by taherv** [comment]
Awesome, I'll wait for the update then.

In the meantime, you can close this bug right now, or when you post updated code .....

**Comment by dblock** [comment]
+1 on fixing this

**Comment by lucasbru** [comment]
+1

**Comment by piotrmurach** [comment]
Fixed and will be part of `v0.7.0` release.
